Gordon360/Controllers/ProfilesController.cs

```diff
@@ -452,6 +452,20 @@ public async Task<ActionResult<FacultyStaffProfileViewModel>> UpdateOfficeLocati
             return Ok(result);
         }
 
+        /// <summary>
+        /// Update office hours
+        /// </summary>
+        /// <param name="value">office hours</param>
+        /// <returns></returns>
+        [HttpPatch]
+        [Route("office_hours/{value}")]
+        public async Task<ActionResult<FacultyStaffProfileViewModel>> UpdateOfficeHours(string value)
```

Comment:
I think the office hours value should not be in the URL. It should be in the body of the request instead.

Reply:
Ditto on this. The current implementation/future one could take the value from the body of the endpoint.
The solution is as simple as removing `{value}` from the Route label.

Reply:
That makes perfect sense😂 Don't know why we have it in the first place.. Just fixed it